pkg/swarm/swarm.go

```diff
@@ -161,9 +161,32 @@ type Chunk interface {
 	Equal(Chunk) bool
 }
 
+// BatchID type represents unique batch identifier.
+type BatchID []byte
```

Comment:
really nice! we should encourage more proper use of go types, thanks

Reply:
Thanks!

Yes, I feel that project would benefit form using types and types aliases. This would lead to better types safety and better code readability.